### PR TITLE
[NEXUS-8906] realms functional tests

### PR DESCRIPTION
**Pull request by mcculls**
https://issues.sonatype.org/browse/NEXUS-8906
http://bamboo.s/browse/NX3-OSSF513-2 :white_check_mark:

### CODE REVIEW

File: testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_realms.t.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests security realms.
+ */
+StartTest(function(t) {
+  t.describe('Security Realms', function(t) {
+    t.it('Realms view is populated', function(t) {
```
Comment (adreghiciu) [inline testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_realms.t.js]: The test seems to only check that the view is shown but description says to me that the test should look that there are realms

**Comment by adreghiciu** [comment]
+1

File: testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_realms.t.js (hunk 1 above)
Comment (mcculls) [inline testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_realms.t.js]: ack: a656a3f

**Comment by kellyrob99** [comment]
+1

**Comment by mcculls** [comment]
Merged to master as d0d7045

**Pull request closed without merging**